webserver/views/api/legacy.py

```diff
@@ -66,6 +72,22 @@ def submit_low_level(mbid):
     return jsonify({"message": "ok"})
 
 
+def map_class_labels(high_level_json):
```

Comment:
why do you have this method twice? It doesn't look like you call this one at all